src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4742,7 +4742,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>The return expression must be of type '{0}' because this method returns by reference</value>
   </data>
   <data name="ERR_CloseUnimplementedInterfaceMemberWrongRefReturn" xml:space="preserve">
-    <value>'{0}' does not implement interface member '{1}'. '{2}' cannot implement '{1}' because it does not return by {3}</value>
+    <value>'{0}' does not implement interface member '{1}'. '{2}' cannot implement '{1}' because it does not have the matching return type reference signature.</value>
```

Comment:
Sounds good.

---
In reply to: [104703783](https://github.com/dotnet/roslyn/pull/17547#discussion_r104703783) [](ancestors = 104703783)